core/src/main/scala/akka/persistence/cassandra/snapshot/CassandraSnapshotStore.scala

```diff
@@ -195,12 +196,18 @@ class CassandraSnapshotStore(cfg: Config) extends SnapshotStore
   }
 
   override def deleteAsync(persistenceId: String, criteria: SnapshotSelectionCriteria): Future[Unit] = {
-    preparedSelectSnapshotMetadata.flatMap { prepStmt =>
-      metadata(prepStmt, persistenceId, criteria, limit = None).flatMap { mds =>
-        val boundStatements = mds.map(md => preparedDeleteSnapshot.map(_.bind(md.persistenceId, md.sequenceNr: JLong)))
-        Future.sequence(boundStatements).flatMap { stmts =>
-          executeBatch(batch => stmts.foreach(batch.add))
+    preparedSelectSnapshotMetadata.flatMap { snapshotMetaPs =>
+      // this meta query gets slower than slower if snaphosts are deleted
```

Comment:
should this comment continue with something " ... and therefore it's recommended to define criteria.minSequenceNr"

Reply:
:+1: